modules/atomic-widgets/widgets/atomic-heading.php

```diff
@@ -32,8 +32,13 @@ protected function render() {
 		$escaped_tag = Utils::validate_html_tag( $settings['tag'] );
 		$escaped_title = esc_html( $settings['title'] );
 
+		$class = '';
+		if ( ! empty( $settings['classes'] ) ) {
+			$class = "class='" . $settings['classes'] . "'";
+		}
+
 		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
-		echo "<$escaped_tag>$escaped_title</$escaped_tag>";
+		echo "<$escaped_tag $class>$escaped_title</$escaped_tag>";
```

Comment:
```suggestion
		echo "<$escaped_tag$class>$escaped_title</$escaped_tag>";
```

remove the space here and add it to the `$class` variable
maybe you should change it to `$attributes`?

Reply:
NO, it will be replaced when we will have a proper solution, it's temporary.